index.html

```diff
@@ -30,12 +30,12 @@
                 <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                     <span class="sr-only">Toggle navigation</span> Menu <i class="fa fa-bars"></i>
                 </button>
+                <a href="http://fossasia.org" id="light-logo" class="link-logo"><img src="img/fossasia-logo-light.png" alt="fossasia logo" class="logo-navbar" style="cursor: pointer;"></a>
+                <a href="http://fossasia.org" id="dark-logo" class="link-logo" style="display: none;margin-left:10px"><img src="img/fossasia-logo.png" alt="fossasia logo" class="logo-navbar" style="cursor: pointer;"></a>
             </div>
             <!-- Collect the nav links, forms, and other content for toggling -->
             <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-                <a href="http://fossasia.org" id="light-logo" class="link-logo"><img src="img/fossasia-logo-light.png" alt="fossasia logo" class="logo-navbar" style="cursor: pointer;"></a>
-                <a href="http://fossasia.org" id="dark-logo" class="link-logo" style="display: none;"><img src="img/fossasia-logo.png" alt="fossasia logo" class="logo-navbar" style="cursor: pointer;"></a>
-                <ul class="nav navbar-nav navbar-right">
+                  <ul class="nav navbar-nav navbar-right">
```

Comment:
Can you de-intend it 2 spaces?

Reply:
@abishekvashok yes. I will do it and commit it again.

Reply:
@abishekvashok I did not get you properly. which portion do I need to intend. Please guide me.

Reply:
he meant indent. remove the indent so the codes start from the same block.